core/middleware.py

```diff
@@ -3,9 +3,34 @@
 from django.http import HttpRequest
 from django.urls import resolve
 from django.utils.deprecation import MiddlewareMixin
+from django_prometheus.middleware import (
+    Metrics,
+    PrometheusAfterMiddleware,
+    PrometheusBeforeMiddleware,
+)
 
 from utils.services import get_long_service_name
 
+# Prometheus metrics that will additional be annotated with User-Agent http header as label
+USER_AGENT_METRICS = [
```

Comment:
not sure what is the best way to choose these metrics is. I went through the list here, and chose the ones that didn't get an error: link[https://github.com/korfuri/django-prometheus/blob/e96708873c7aaf49f5b571cb08804aaecb45cc9d/django_prometheus/middleware.py#L24](url)

Reply:
django_prometheus likes to error when you assign a label to a metric and the corresponding label value does not get populated